examples/cifar/train_cifar_custom_loop.py

```diff
@@ -100,20 +109,20 @@ def main():
             print('epoch: ', train_iter.epoch)
             print('train mean loss: {}, accuracy: {}'.format(
                 sum_loss / train_count, sum_accuracy / train_count))
-            # evaluation
+            # Evaluate the model.
             sum_accuracy = 0
             sum_loss = 0
-            model.predictor.train = False
-            for batch in test_iter:
-                x_array, t_array = convert.concat_examples(batch, args.gpu)
-                x = chainer.Variable(x_array)
-                t = chainer.Variable(t_array)
-                loss = model(x, t)
-                sum_loss += float(loss.data) * len(t.data)
-                sum_accuracy += float(model.accuracy.data) * len(t.data)
+            # It is good practice to turn off train mode during evaluation.
+            with configuration.using_config('train', False):
```

Comment:
Oh, is it a severe bug?

Reply:
Yes, without this, layers such as dropout will not work correctly in test mode. Since it may take a while longer for this PR, shall I submit a separate bug-fix PR?